File: js/isPrime.js
let num = "31";
let count = 0;


for (let i = 0; i * i < num; i++) {
    if (num % i == 0) {
        count++;
        if (num / i != i)
            count++;
    }

}
console.log(count);
if (count == 2)
    console.log("This is a Prime Number");
else
    console.log("This is Not Prime Number");

File: CPP/isPrime.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    int n, i;
    bool isPrime = true;
    cin >> n;
    for (i = 2; i < n / 2; i++)
    {
        if (n % i != 0)
        {
            continue;
        }
        else
        {
            isPrime = false;
            break;
        }
    }
    if (isPrime)
        cout << "yes, It is a Prime Number";
    else
        cout << "No, Its Not a Prime Number";
}